### PR TITLE
Loading page for Oauth login in android

### LINKED ISSUE #1167: Android - Oauth logins don't have loading page
Aouth logins dont have loading page that produces unexpected behavior

### PR DESCRIPTION
**Pull request by RodriSanchez1**
close #1167

### CODE REVIEW

File: src/components/WelcomeScreen/WelcomeScreen.container.js [hunk 1]
```diff
@@ -61,13 +61,9 @@ export class WelcomeScreen extends Component {
           offline: true // optional, but requires the webClientId - if set to true the plugin will also return a serverAuthCode, which can be used to grant offline access to a non-Google server
         },
         function(obj) {
-          login(
```
Comment (tomivm) [inline src/components/WelcomeScreen/WelcomeScreen.container.js:64]: Login action now is never used in this file. Remove from import and mapDispatchToProps

File: src/components/Account/OAuthLogin/OAuthLogin.container.js [hunk 2]
```diff
@@ -54,11 +60,20 @@ class OAuthLoginContainer extends React.Component {
     return !!user.email;
   }
 
+  handleError() {
+    const { history } = this.props;
+    this.setState({ hasErrors: true });
+    setTimeout(() => {
```
Comment (tomivm) [inline src/components/Account/OAuthLogin/OAuthLogin.container.js:65]: for me. is not necessary to stop the user interface to show the error message. probably we could redirect to /login-signup directly and show the error message on a notification or modal.

**Comment by martinbedouret** [comment]
@RodriSanchez1 please rewrite PR title

File: src/components/Account/OAuthLogin/OAuthLogin.container.js [hunk 3]
```diff
@@ -54,11 +59,20 @@ class OAuthLoginContainer extends React.Component {
...
+      history.replace('/');
```
Comment (tomivm) [inline src/components/Account/OAuthLogin/OAuthLogin.container.js]: Probably would be better to redirect the user to /login-signup if an error occurred

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**